core/test_network/src/lib.rs

```diff
@@ -58,7 +58,7 @@ pub fn get_key_pair() -> KeyPair {
         .expect("Public key not in mulithash format"),
         PrivateKey::from_hex(
             Algorithm::Ed25519,
-            "9AC47ABF 59B356E0 BD7DCBBB B4DEC080 E302156A 48CA907E 47CB6AEA 1D32719E 7233BFC8 9DCBD68C 19FDE6CE 61582252 98EC1131 B6A130D1 AEB454C1 AB5183C0",
+            "9AC47ABF 59B356E0 BD7DCBBB B4DEC080 E302156A 48CA907E 47CB6AEA 1D32719E 7233BFC8 9DCBD68C 19FDE6CE 61582252 98EC1131 B6A130D1 AEB454C1 AB5183C0".as_ref(),
```

Comment:
Should we consider applying this formatting to other hard-coded keys?

Reply:
you mean uppercase space-separated?

Reply:
I'll do it in a separate PR

Reply:
opened [#3316](https://github.com/hyperledger/iroha/pull/3316)